invenio_github/ext.py

```diff
@@ -52,6 +52,13 @@ def release_api_class(self):
         assert issubclass(cls, GitHubRelease)
         return cls
 
+    @cached_property
+    def release_error_handlers(self):
+        """Github Release error handlers."""
+        error_handlers = current_app.config.get('GITHUB_ERROR_HANDLERS', [])
+        return [(import_string(error_cls), import_string(handler))
```

Comment:
Instead of directly `import_string`, you could add the [`obj_or_import_string` utility]( https://github.com/inveniosoftware/invenio-stats/blob/3d6ef0afc21bc7cb03339dce128c2150e5adc011/invenio_stats/utils.py#L93-L104), which also better handles direct objects/functions being passed.